tests/ci/run_minimal_tests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+source tests/ci/common_posix_setup.sh
+
+# build AWS-LC without tests and ssl module.
+echo "Building AWS-LC without building tests code and ssl module in debug mode."
+run_build -DBUILD_TESTING=0 -DBUILD_SSL=0
+
+echo "Building AWS-LC without building tests code and ssl module in release mode."
+run_build -DBUILD_TESTING=0 -DBUILD_SSL=0 -DCMAKE_BUILD_TYPE=Release
```

Comment:
This should be `-DBUILD_LIBSSL` instead of `-DBUILD_SSL`.

nit: Change to be `-DBUILD_TESTING=OFF` and `-DBUILD_LIBSSL=OFF`

Reply:
Fixed.